.clang-tidy

```diff
@@ -26,9 +26,18 @@ Checks:     '
             performance-*,
             portability-*,
             readability-*,
+            -bugprone-exception-escape,
+            -bugprone-macro-parentheses,
+            -bugprone-too-small-loop-variable,
+            -clang-analyzer-cplusplus.NewDelete,
+            -clang-analyzer-cplusplus.NewDeleteLeaks,
             -google-readability-braces-around-statements,
-            -google-readability-namespace-comments,
+            -modernize-use-nodiscard,
+            -modernize-avoid-c-arrays,
+            -performance-unnecessary-value-param,
             -readability-braces-around-statements,
+            -readability-isolate-declaration,
```

Comment:
Doesn't let you do:
`uint8_t x, y;`
I don't really like that, but am open to a conversation about it.

Reply:
I also don't like that, but would approve without further comment or discussion, so..